presto-benchmark-runner/src/test/java/com/facebook/presto/benchmark/source/TestDbBenchmarkSuiteSupplier.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.source;
+
+import com.facebook.presto.testing.mysql.TestingMySqlServer;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.Jdbi;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.facebook.airlift.testing.Closeables.closeQuietly;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.BENCHMARK_QUERIES_TABLE;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.BENCHMARK_SUITES_TABLE;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.XDB;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.getBenchmarkSuiteObject;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.getJdbi;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.insertBenchmarkQuery;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.insertBenchmarkSuite;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.setupMySql;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.truncateBenchmarkQueries;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.truncateBenchmarkSuites;
+import static org.testng.Assert.assertEquals;
+
+public class TestDbBenchmarkSuiteSupplier
```

Comment:
You need to annotate the class with `@Test(singleThreaded=true)`, to avoid test methods running in parallel, because all the methods are sharing a single MySQL instance and they'll interfere with each other.